docs/java/guides/cf-xsuaa.md

```diff
@@ -0,0 +1,229 @@
+---
+id: cf-xsuaa
+title: SAP Cloud Foundry
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Cloud Foundry XSUAA Usage
+custom_edit_url: https://github.com
+description: Configure you Cloud Foundry CLI and bind it to SAP Cloud Platform
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+- xsuaa
+- destination
+- connectivity
+image:
+---
+
+## XSUAA Usage on Cloud Foundry
+
+When implementing and deploying an application it quickly becomes important to understand how authentication and authorizations works on SCP Cloud Foundry.
+In our tutorials and documentation pages we recommend using "approuter" as proxy service to handle authentication management to your implemented application.
+The following steps will show what happens behind the scenes. The requests can be manually reproduced by a REST client of your choice, e.g. Postman or Insomnia.
+
+The [official UAA documentation](https://docs.cloudfoundry.org/api/uaa/) is highly recommended if there are questions.
+
+**Please note:** The following paragraphs only describe a subset of features of the XSUAA Service on Cloud Foundry.
+The Cloud SDK is not participating in the active development of this software and cannot give in-depth support.
+Some information may already be outdated.
+
+
+These are use cases described below:
+- User Login: `Authorization Code Grant`
+- SCP Service Usage on behalf of User: `JWT Bearer Token Grant`
+- SCP Service Usage on behalf of service: `Client Credentials Grant`
+- Resolve User Access Token: `Refresh Token Grant`
+
+### Read the Application Properties  
+
+In order to create a request, we need to parse the XSUAA connection data.
+
+1. First, take note of your "_application route_", that's the URL for which an authorization request will be constructed.
+
+1. Open the **system-provided** environment variables of your application on Cloud Foundry.
+
+1. In the JSON value, locate the object `VCAP_SERVICES.xsuaa[0].credentials`.
+   
+   *Note:* Depending on the scenario, the "xsuaa" array may have more than one entry. Your application can be bound to multiple instances, e.g. through different service plans.
+
+1. Extract values "_url_", "_clientid_", "_clientsecret_".
+
+## Authorization Code Grant
+
+Since we start without previous access token, we begin our journey with the browser flow of [Authorization Code Grant](https://docs.cloudfoundry.org/api/uaa/version/74.23.0/index.html#authorization-code-grant).
```

Comment:
```suggestion
Since we start without an existing access token, our journey begins with the browser flow of [Authorization Code Grant](https://docs.cloudfoundry.org/api/uaa/version/74.23.0/index.html#authorization-code-grant).
```